README.md

```diff
@@ -3,4 +3,356 @@
+        <allowRead>true</allowRead>
+        <modifyAllRecords>false</modifyAllRecords>
+        <object>Account</object>
+        <viewAllRecords>false</viewAllRecords>
+    </objectPermissions>
+</PermissionSet>
+```
+
+Upload this permission set to your org:
+
+```sh
+sf deploy metadata --ignore-conflicts
+```
+
+Then assign the permissions to the `Functions` profile with
+
+```sh
+sfdx force:user:permset:assign -n Functions
+```
+
+## Create and Run a Python Function Locally
+
+### Generate the Python Function
+
+From the SFDX project root, run:
+
+```sh
+sf generate function \
+  --language python \
+  --function-name hellopython
+```
+
+The remaining commands will be executed within the newly generated project folder. Change to that folder by running:
+
+```sh
+cd functions/hellopython
+```
+
+You should also create a `.gitignore` file in the function directory with the following contents:
+```gitignore
+venv
+__pycache__
+```
+
+The following command will do this for you:
+```sh
+echo "venv\n__pycache__" > .gitignore
+```
+
+### Create the Python [Virtual Environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) & Install Dependencies
+
+To install the dependencies required by the Python Function locally, we first need to create a "Virtual Environment" (venv) 
+which we can install packages into without affecting your system Python installation.  This can be done by running:
+```sh
+python3 -m venv venv
+```
+
+Next, the virtual environment needs to be activated.  
+
+On a **macOS / Linux system** you can activate the virtual environment with
+
+```sh
+source venv/bin/activate
+```
+
+On a **Microsoft Windows system** you can activate the virtual environment with
+```sh
+.\venv\Scripts\activate
+```
+
+> For help with setting up a virtual environment, see the [Python documentation](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment).
+
+Finally, the dependencies can be installed into the newly created environment:
+
+```sh
+pip3 install -r requirements.txt
+```
+
+
+This will ensure your function has all dependencies it requires installed before you run it.  If you forgot to do this 
+before starting the Function locally you’ll receive an error reminding you to perform this step.
+
+### Run the Python Function Locally
+
+```sh
+sf run function start
+```
+
+This will start the function running locally on http://localhost:8080.  Messages logged by the running function will appear here.
+
+### Invoke the Running Python Function
+
+After starting the function:
+
+* open a new command line terminal 
+* navigate to the `hellopython` directory 
+* invoke the function by sending it a payload
+
+```sh
+sf run function --function-url http://localhost:8080 --payload '{}'
+```
+
+## Deploy the Python Function
+
+The following commands need to be executed from the **root of your SFDX project** so change into that directory now.
```

Comment:
I suspect some people might miss this mention (I did the first time through).
I think perhaps adding the `cd ../../` before the `git add` might make it harder to miss?

Reply:
Addressed in #34.